app/imports/startup/slingshot.js

```diff
@@ -0,0 +1,39 @@
+/* globals Slingshot Random */
+import { Meteor } from 'meteor/meteor';
+import { Random } from 'meteor/random';
+
+Slingshot.fileRestrictions('imageUpload', {
+  allowedFileTypes: ['image/png', 'image/jpeg', 'image/gif'],
```

Comment:
but as we don't want anything other than a recently taken photo why not ban the other formats?